docs/library/processors/panning/stereobalance/example-0.py

```diff
@@ -0,0 +1,22 @@
+from signalflow import *
+graph = AudioGraph()
+
+#-------------------------------------------------------------------------------
+# Demonstrating the effects of StereoBalance. Two tones are first panned hard
+# left and hard right with StereoPanner. 
+# Setting StereoBalance's balance value to 0.0 will mean both tones are heard 
+# equally. A value of -1.0 will result in only the left channel being heard. 
+# A value of 1.0 will result in only the right channel being heard.
+#-------------------------------------------------------------------------------
+
+low = TriangleOscillator(220)
+high = TriangleOscillator(660)
+
+left = StereoPanner(low, -1.0)
+right = StereoPanner(high, 1.0)
+panned = (left + right)
+
+balanced = StereoBalance(panned, -1.0)
```

Comment:
The panning code here could be simplified by just using an array [low, high]. I also think it would be nice to animate the output so the listener gets a sense of what is being changed. Something like this?

```
low = TriangleOscillator(220)
high = TriangleOscillator(660)

panned = ChannelArray([low, high])

balanced = StereoBalance(panned, TriangleLFO(1, -1, 1))
```

Reply:
Good points -- I think that it could be worth having a discussion about what and who should be prioritised in the examples?

E.g. are we writing examples for people who are used to programming in Python, and want to get up and running quickly with SignalFlow? In which case writing more concise and simplified examples would be a good idea. 

Or are we writing examples for people who are perhaps new to programming in Python or programming in general? In which case I think it makes sense to be a bit more verbose and therefore more explicit about what is going on, breaking things down into clear steps. (This is what I have been leaning towards so far).

Would be great to hear any thoughts on what the balance (no pun intended) should be!

Reply:
I'd say the first - people who are familiar with Python and want to get up and running quickly. The key thing in my mind is to demonstrate best practices, hence using the abbreviated array notation (rather than summing two StereoPanners), as it's (i) quicker and more usable when you're doing a lot of prototyping, and (ii) more efficient CPU-wise than summing StereoPanners. The array notation to denote multichannel is definitely one of the "new and unique" things to SignalFlow, but it's also one of the core features that people will need to become familiar with to become an effective practitioner.

Maybe a good middle-ground is to include one-liner comments above non-obvious lines (e.g. the ChannelArray above) that briefly explains what's happening?
```
# Create a stereo pair with `low` on the left and `high` on the right
```